doc/UsersGuide/source/CodeOverview.rst

```diff
@@ -20,103 +54,107 @@ The UFS Weather Model (:term:`WM`) repository supports the :term:`UFS` short- an
+    │     └── CDEPS
+    │         ├── (datm)                         -------- CDEPS DATM
+    │         └── (docn)                         -------- CDEPS DOCN
+    ├── CICE-interface
+    │    └── CICE                                -------- CICE6 sea ice model
+    │        ├── (icepack)                       -------- Sea ice column physics
+    │        └── (cicecore/drivers/nuopc/cmeps)  -------- NUOPC CICE6 cap
+    ├── cmake                                    -------- cmake configuration files
+    ├── CMakeLists.txt         
+    ├── CMakeModules           
+    ├── CMEPS-interface
+    │    └── CMEPS
+    │         └── (cesm)                         -------- CMEPS CESM
+    ├── doc                                      -------- User Guide files
+    ├── driver                 
+    ├── FV3                                      -------- UFSAtm atmosphere model
+    │   ├── (atmos_cubed_sphere)                 -------- FV3 dynamical core
+    │   │   ├── (docs)
+    │   │   ├── (driver)
+    │   │   ├── (model)
+    │   │   └── (tools)
+    │   ├── (ccpp)                               -------- Common Community Physics Package
+    │   │   ├── (config)
+    │   │   ├── (driver)
+    │   │   ├── (framework)                      -------- CCPP framework
+    │   │   ├── (physics)                        -------- CCPP-compliant physics schemes
+    │   │   └── (suites)                         -------- CCPP physics suite definition files (SDFs)
+    │   ├── (cpl)                                -------- Coupling field data structures
+    │   ├── (io)                                 -------- UFSAtm write grid comp code
+    │   └── (stochastic_physics)                 -------- Wrapper for stochastic physics
+    ├── GOCART
+    │    └── (ESMF)                              -------- GOCART model
+    ├── HYCOM-interface
+    │    └── HYCOM                               -------- HYCOM ocean model
+    │        └── (NUOPC)                         -------- NUOPC HYCOM cap
+    ├── LICENSE.md
+    ├── modulefiles                              -------- system module files for supported HPC systems
+    ├── MOM6-interface
+    │    └── MOM6
+    │        ├── (src)                           -------- MOM6 ocean model
+    │        └── (config_source/drivers/nuopc_cap)  -------- NUOPC MOM6 cap
+    ├── NOAHMP-interface
+    │    └── noahmp
+    │        ├── (cmake)                         -------- Noah-MP land model
+    │        ├── (drivers/nuopc)                 -------- NUOPC Noah-MP cap
+    │        ├── (parameters)
+    │        └── (src)
+    ├── README.md
+    ├── stochastic_physics                       -------- stochastic physics pattern generator
+    ├── tests                                    -------- regression test infrastructure
+    │   └── parm
+    │   └── tests
+    │   └── fv3_conf   
+    └── WW3
+         └── (model)                             -------- WW3 model
+             └── (esmf)                          -------- NUOPC WW3 cap
```

Comment:
We probably need a modification here also. There are now two WW3 caps, but neither resides in a directory called esmf. They reside in a (src) subdirectory. Maybe
 
└── (src) -------- NUOPC WW3 caps

Reply:
@DeniseWorthen Updated! There are a LOT of files in the (src) directory. Do you think there are other ones we should mention besides the caps?

Reply:
Looking at the other components, we're generally just pointing to the directory containing the cap. It is just in the WW3 case, the cap resides in the same directory as all the other code.